index.js

```diff
@@ -64,18 +63,34 @@ async function getCurrentRelease(app) {
     await exec.exec(`gigalixir releases -a ${app}`, [], options);
   });
 
-  const currentRelease = Number(JSON.parse(releasesOutput)[0].version);
+  const releases = JSON.parse(releasesOutput);
+  // first release number in gigalixir is 1
+  const currentRelease = releases.length && Number(releases[0].version) || 0;
 
   return currentRelease;
 }
 
+function printRelease(releaseNumber) {
+  return releaseNumber || "[NO PREV RELEASE]"
+}
+
 async function run() {
   try {
-    const gigalixirUsername = core.getInput('GIGALIXIR_USERNAME', { required: true });
-    const gigalixirPassword = core.getInput('GIGALIXIR_PASSWORD', { required: true });
-    const sshPrivateKey = core.getInput('SSH_PRIVATE_KEY', { required: true });
-    const gigalixirApp = core.getInput('GIGALIXIR_APP', { required: true });
-    const migrations = core.getInput('MIGRATIONS', { required: true });
+    const gigalixirUsername = core.getInput('GIGALIXIR_USERNAME', {
+      required: true
+    });
+    const gigalixirPassword = core.getInput('GIGALIXIR_PASSWORD', {
+      required: true
+    });
+    const sshPrivateKey = core.getInput('SSH_PRIVATE_KEY', {
+      required: true
+    });
+    const gigalixirApp = core.getInput('GIGALIXIR_APP', {
+      required: true
+    });
+    const migrations = core.getInput('MIGRATIONS', {
+      required: true
+    });
```

Comment:
Can you extract `{required: true}` to a variable called `baseInputOptions` and pass that into these?

e.g., `const gigalixirUsername = core.getInput('GIGALIXIR_USERNAME, baseInputOptions)`